config/config.exs

```diff
@@ -65,6 +65,8 @@ config :kafka_ex,
     certfile: File.cwd!() <> "/ssl/cert.pem",
     keyfile: File.cwd!() <> "/ssl/key.pem"
   ],
+
+  snappy_module: :snappyer,
```

Comment:
This makes it so that the default in dev will be snappyer, and a lot of people will just copy this into their own projects which will make the documentation confusing for them.  Please either change it to `:snappy` it or update the documentation.

Reply:
The doc is changed.